examples/webgl_nodes_playground.html

```diff
@@ -82,19 +82,19 @@
 
 				// Lights
 
-				const topLight = new THREE.PointLight( 0xF4F6F0, 1 );
+				const topLight = new THREE.PointLight( 0xF4F6F0, 5, 60000 );
 				topLight.position.set( 0, 1000, 1000 );
 				scene.add( topLight );
 
-				const backLight = new THREE.PointLight( 0x0c1445, 1 );
+				const backLight = new THREE.PointLight( 0x0c1445, 3, 60000 );
 				backLight.position.set( - 100, 20, - 260 );
 				scene.add( backLight );
 
 				renderer = new THREE.WebGLRenderer( { antialias: true } );
 				document.body.appendChild( renderer.domElement );
 				renderer.outputEncoding = THREE.sRGBEncoding;
 				renderer.toneMapping = THREE.LinearToneMapping;
-				renderer.toneMappingExposure = 4000;
+				renderer.toneMappingExposure = 900000;
```

Comment:
I noticed this previously... I'm still wondering why your `exposure` values are so huge... 🤔

Reply:
Distances involved on the order of kilometers, light intensities on the order of small candles, this may be a problem. :) 

@sunag note that in physically-correct mode the light's distance is simply a range cutoff. It doesn't affect intensity over most of the light's range, so that part of this change may not be having the effect you want.

Ideally — I would scale the scene down by a factor of 1000 to something more like an indoor scene, increase light intensities to the low thousands, and then choose an exposure somewhere in the vicinity of 2<sup>-5</sup> to 2<sup>+5</sup>.

Probably a good time for us to get going on that migration guide! 😅

Reply:
Sorry, these settings are very artistic 😅

Reply:
I will follow this parameters for the rest of the examples added here, what do you think? @donmccurdy @WestLangley 
https://github.com/mrdoob/three.js/pull/24981/commits/fa92b22347c95fd9849c5bb662279f46a014e281

Reply:
That's good with me, thank you!

Reply:
Thanks to you :)